apps/src/aichat/types.ts

```diff
@@ -3,24 +3,48 @@ import {LevelProperties} from '@cdo/apps/lab2/types';
 // TODO: Update this once https://codedotorg.atlassian.net/browse/CT-471 is resolved
 export type AichatInteractionStatusValue = string;
 
-export type ChatCompletionMessage = {
-  id: number;
-  role: Role;
+export interface ChatItem {
+  // UTC timestamp in milliseconds
+  timestamp: number;
+}
+
+export interface ChatMessage extends ChatItem {
   chatMessageText: string;
+  role: Role;
   status: AichatInteractionStatusValue;
-  chatMessageSuffix?: ChatMessageSuffix;
-  timestamp?: string;
-};
+}
 
-export type AichatCompletionMessage = Pick<
-  ChatCompletionMessage,
-  'role' | 'chatMessageText' | 'status'
->;
+export interface ModelUpdate extends ChatItem {
+  id: number;
```

Comment:
Maybe overkill, but any value in `Notification` and `ModelUpdate` both being derived from a type that has an `id` (`ChatItemRemovable`, since that's why we add an id field)?

Reply:
Yeah good idea, I think as a next step I wanted to look into component-izing the actual chat UI components, in which case I can see `ModelUpdate` and `Notification` being rendered by some single `ChatNotification` component that would have a common interface. So might make sense to do it then?